tests/test_judge_repairer.py

```diff
@@ -319,15 +328,15 @@ def change_availability(new_value):
 
             if ignore_availability:
                 change_availability(False)
-                rule_id = add_rule(dids=[{'scope': scope, 'name': dataset}], account=self.jdoe, copies=1, rse_expression=rse, grouping=grouping, weight=None, lifetime=None, locked=False, subscription_id=None, ignore_availability=ignore_availability, activity='DebugJudge')[0]
+                rule_id = add_rule(dids=[{'scope': scope, 'name': dataset}], account=self.jdoe, copies=1, rse_expression=rse, grouping=grouping, weight=None, lifetime=None, locked=False, subscription_id=None, ignore_availability=ignore_availability, activity='DebugJudge')[0]  # NOQA: E501
```

Comment:
It would be better to break this line down into multiple lines instead of adding `NOQA: E501`

Reply:
Changed, now the arguments are on separate lines